src/rabbit_msg_store_ets_index.erl

```diff
@@ -68,8 +68,8 @@ delete_object(Obj, State) ->
     true = ets:delete_object(State #state.table, Obj),
     ok.
 
-delete_by_file(File, State) ->
-    MatchHead = #msg_location { file = File, _ = '_' },
+cleanup_undefined_file(State) ->
```

Comment:
I think that's an implementation detail that can change over time. "without_file" is entirely generic.